src/components/tag-input.js

```diff
@@ -38,7 +38,16 @@ export default class TagInput extends LitElement {
   afterPaste(e) {
     const clipboardData = e.clipboardData || window.clipboardData;
     const pastedData = clipboardData.getData('Text');
-    this.value = pastedData ? pastedData.split(',').filter((v) => v.trim() !== '') : '';
+    const pastedArray = pastedData ? pastedData.split(',').filter((v) => v.trim() !== '') : '';
+    
+    if(pastedArray) {
```

Comment:
seems like `if ... else` is not as per the style guide. but otherwise fine (can be fixed at our end)

Reply:
will fix at our end